src/main/Port.Adapter/In/InProcess/In.InProcess.nuspec

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package >
   <metadata>
-    <id>ei8.Data.ExternalReference.Port.Adapter.In.InProcess</id>
+    <id>ei8.Data.Mirror.Port.Adapter.In.InProcess</id>
     <version>0.1.0</version>
-    <title>ei8.Data.ExternalReference.Port.Adapter.In.InProcess</title>
+    <title>ei8.Data.Mirror.Port.Adapter.In.InProcess</title>
     <authors>Elmer Roderick Niem Bool</authors>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <licenseUrl>https://www.gnu.org/licenses/agpl-3.0.en.html</licenseUrl>
```

Comment:
Just need to rename the "external reference" in the "description" and "tags" to "mirror".